app/login.go

```diff
@@ -70,5 +72,16 @@ func (a *App) Login(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if res.ChallengeName != nil && *res.ChallengeName == "NEW_PASSWORD_REQUIRED" {
+		params = map[string]*string{
+			"username": aws.String(username),
+			"session":  res.Session,
+		}
+		b, _ := json.Marshal(params)
+		s := url.QueryEscape(string(b))
+		http.Redirect(w, r, fmt.Sprintf("/new_password_required?params=%s", s), http.StatusFound)
```

Comment:
That's a good idea, but I think the `data` object here is too simple to justify usage of `encoding/json` and `net/url`. Since go is about simplicity, could we do this instead?
```suggestion
		http.Redirect(w, r, fmt.Sprintf("/new_password_required?username=%s&session=%s", username, res.Session), http.StatusFound)
```